entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityHandlerTests.kt

```diff
@@ -376,17 +430,25 @@ class EntityHandlerTests {
         val entityId = "urn:ngsi-ld:BreedingService:0214"
 
         every { entityService.exists(any()) } returns true
-        every { entityService.appendEntityAttributes(any(), any(), any()) } throws BadRequestDataException("@type not found")
+        every {
+            entityService.appendEntityAttributes(
+                any(),

```

Comment:
The hard wrap is for any method call?

Reply:
It is not. This one is a consequence of the hard wrap of the whole line : 
```kotlin
every { entityService.appendEntityAttributes(any(), any(), any()) } throws BadRequestDataException("@type not found")
``` 
Could be : 
```kotlin
every { entityService.appendEntityAttributes(any(), any(), any()) }
    throws BadRequestDataException("@type not found")
```
or : 
```kotlin
every { 
    entityService.appendEntityAttributes(any(), any(), any()) 
} throws BadRequestDataException("@type not found")
```
But Intellij seems to apply formatting rules for long lines to every elements in the line.